bin/build

```diff
@@ -5,16 +5,40 @@ set -o pipefail
 layers_dir=$1
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 echo -e 'launch = true' > $layers_dir/system-function.toml
+node_modules_dir=$layers_dir/node_modules
```

Comment:
This gets nested under its own folder right? I wouldn't want to accidentally pollute the nodejs-engine/nodejs-npm's folders.

Reply:
Correct.  It should be happening in its own layer e.g. `heroku_evergreen-node-system-function`.

Reply:
I just verified that it is indeed creating and storing the cached `node_modules` under `/layers/heroku_evergreen-node-system-function/node_modules`

Reply:
@jabrown85 I'm double checking one other thing.  I need to make sure I'm exploding the `node_modules` directory correctly from the cache to the system function layer.

Reply:
@jabrown85 should be gtg.  Note to self, don't forget to remove the local `node_modules` directory when doing local buldpack testing.